src/BioFSharp.Mz/ProteinInference.fs

```diff
@@ -30,6 +36,68 @@ module ProteinInference =
             Class: PeptideEvidenceClass
         }    
 
+    /// For a group of proteins, contains information about all peptides that might be used for its quantification and score calculated for it.
+    type InferredProteinClassItemScored =
+        {
+            GroupOfProteinIDs: string
+            PeptideSequence  : string []
+            Class            : PeptideEvidenceClass
+            TargetScore      : float
+            DecoyScore       : float
+            Decoy            : bool
+            DecoyBigger      : bool
+            FoundInDB        : bool
+        }
+
+    /// For a group of proteins, contains information about all peptides that might be used for its quantification and score / q-value calculated for it.
+    type InferredProteinClassItemQValue =
```

Comment:
this type has only one additional field, could the same functionality be achieved by using a record type with 
two fields, one being "Qvalue" of type float and one being 'Item' of type InferredProteinClassItemScored? I think the readability would gain from this